packages/enketo-core/README.md

```diff
@@ -169,6 +169,12 @@ The `language` option overrides the default languages rules of the XForm itself.
 6. run tests with `yarn test` (headless chrome) and `yarn run test-browsers` (browsers); **note:** running tests updates the coverage badge in README.md, but these changes should not be committed except when preparing a release
 7. adding the querystring `touch=true` and reducing the window size allows you to simulate mobile touchscreens
 
+### Common development setup issue troubleshooting
+
+-   `ERESOLVE unable to resolve dependency tree`: Make sure you're using node >=18 <21. Try [nvm](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating).
+-   `Error: Cannot find module...` and `npm ERR! code ENOWORKSPACES`: This is a monorepo. Make sure you run `yarn install` in the root of this project before moving into `packages/enketo-core`. If your using nvm, make sure you run `nvm use` from the root of this project before moving into `packages/enketo-core`. Running `nvm use` from `packages/enketo-core` will not work.
+-   `EADDRINUSE`: If you get an `EADDRINUSE` error when you `yarn start`, try searching for a ghost node process and killing it.
```

Comment:
Do you have a sense of how you got into this state? Was it because you had an unrelated node process running or was it something about the instructions for Enketo that got you there? I'm wondering because searching for "EADDRINUSE node" seems like a good way to get the needed information so maybe it doesn't specifically need to be addressed in this readme.